actors/builtin/miner/monies.go

```diff
@@ -3,15 +3,59 @@ package miner
 import (
 	"github.com/filecoin-project/specs-actors/actors/abi"
 	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	"github.com/filecoin-project/specs-actors/actors/builtin"
 )
 
+// IP = InitialPledgeFactor * BR(precommit time)
+var InitialPledgeFactor = big.NewInt(20)
+
+// FF = (DeclaredFaultFactorNum / DeclaredFaultFactorDenom) * BR(t)
+var DeclaredFaultFactorNum = big.NewInt(214)
+
+// FF = (DeclaredFaultFactorNum / DeclaredFaultFactorDenom) * BR(t)
+var DeclaredFaultFactorDenom = big.NewInt(100)
+
+// SP = UndeclaredFaultFactor * BR(t)
+var UndeclaredFaultFactor = big.NewInt(5)
+
+// This is the BR(t) value of the given sector for the current epoch.
+// It is the expected reward this sector would pay out over a one day period.
+// BR(t) = CurrEpochReward(t) * SectorQualityAdjustedPower * EpochsInDay / TotalNetworkQualityAdjustedPower(t)
+func ExpectedDayRewardForSector(epochTargetReward abi.TokenAmount, networkQAPower abi.StoragePower, qaSectorPower abi.StoragePower) abi.TokenAmount {
+	expectedRewardForProvingPeriod := big.Mul(big.NewInt(builtin.EpochsInDay), epochTargetReward)
+	return big.Div(big.Mul(qaSectorPower, expectedRewardForProvingPeriod), networkQAPower)
+}
+
+// This is the FF(t) penalty for a sector expected to be in the fault state either because the fault was declared or because
+// it has been previously detected by the network.
+// FF(t) = DeclaredFaultFactor * BR(t)
+func PledgePenaltyForSectorDeclaredFault(epochTargetReward abi.TokenAmount, networkQAPower abi.StoragePower, qaSectorPower abi.StoragePower) abi.TokenAmount {
```

Comment:
Ah, none of these take info about a single sector! Rename to talk just about power instead, so that it's clear we could re-arrange the arithmetic to sum the power first and then compute fees.